src/cmd/cmd_trim.go

```diff
@@ -61,10 +61,11 @@ func RunTrimCmdLine() {
 	loadSpec.TimeColumn = *flags.TIME_COL
 
 	trimSpec := sybil.TrimSpec{}
+	trimSpec.TimeColumn = *flags.TIME_COL
 	trimSpec.DeleteBefore = int64(*DELETE_BEFORE)
 	trimSpec.MBLimit = int64(*MB_LIMIT)
 
-	toTrim := t.TrimTable(flags, &trimSpec)
+	toTrim := t.TrimTable(&trimSpec)
```

Comment:
note to self: do we have trimming tests?

Reply:
initial coverage check seems to indicate TrimTable is not covered in test.